ray-operator/test/e2eautoscaler/raycluster_autoscaler_test.go

```diff
@@ -253,3 +257,75 @@ func TestRayClusterAutoscalerWithDesiredState(t *testing.T) {
 	g.Expect(err).NotTo(gomega.HaveOccurred())
 	g.Expect(pods).To(gomega.HaveLen(maxReplica))
 }
+
+func TestRayClusterAutoscalerMinReplicasUpdate(t *testing.T) {
+	test := With(t)
+	g := gomega.NewWithT(t)
+
+	// Create a namespace
+	namespace := test.NewTestNamespace()
+	test.StreamKubeRayOperatorLogs()
+
+	// Scripts for creating and terminating detached actors to trigger autoscaling
+	scriptsAC := newConfigMap(namespace.Name, files(test, "create_detached_actor.py", "terminate_detached_actor.py"))
```

Comment:
`terminate_detached_actor.py` is not used in this test, remove it.

Reply:
Fixed.